Librdmacm links against libibverbs, and the current version of libibverbs
uses dlopen(), so librdmacm must also link against libdl.

Upstream-Status: Pending

Signed-off-by: Donn Seeley <donn.seeley@windriver.com>
---
 configure.in |    2 ++
 1 file changed, 2 insertions(+)

--- a/configure.in
+++ b/configure.in
@@ -39,6 +39,8 @@ AC_CHECK_SIZEOF(long)
 dnl Checks for libraries
 AC_CHECK_LIB(pthread, pthread_mutex_init, [],
     AC_MSG_ERROR([pthread_mutex_init() not found.  librdmacm requires libpthread.]))
+AC_CHECK_LIB(dl, dlsym, [],
+    AC_MSG_ERROR([dlsym() not found.  librdmacm requires libdl.]))
 if test "$disable_libcheck" != "yes"; then
 AC_CHECK_LIB(ibverbs, ibv_get_device_list, [],
     AC_MSG_ERROR([ibv_get_device_list() not found.  librdmacm requires libibverbs.]))
